### DISCUSSION TITLE
Add built-in help module (like RoamResearch)

### DISCUSSION THREAD

**Discussion by AoCz** [athensresearch/athens#1373] [Feature Requests]
Adding a built-in help page like Roam's "?" would also be very useful. Right now, with all the new features being implemented in Athens, it can be easy for users to not notice a new feature. Also, having an easily navigable help module with examples and search functionality would go a long way to introducing Athens to new users who are coming more from WYSIWYG apps Microsoft Word.

**Comment by joelhans** [comment]
There is some discussion around a help menu in this issue: https://github.com/athensresearch/athens/issues/649

A few contributors have tried implementation, or are interested in it, but no one has quite gotten to the finish line yet.

If you have more thoughts about what type of content you'd like to see in a future help menu, definitely feel free to leave some notes here or in that issue.
